### PR TITLE
fix wrong link

### PR DESCRIPTION
**Pull request by luffyke**
1. use hugo relref tag fix wrong link

### CODE REVIEW

**Comment by appleboy** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by tboerger** [comment]
~~~Looks like it have not been tested,~~~ The local server results in correct link, but the docs page link results in `https://docs.gitea.io/zh-cn/hacking-on-gitea/zh-cn/installation/from-source`

File: content/doc/advanced/hacking-on-gitea.zh-cn.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ menu:
 
 # Hacking on Gitea
 
-首先你需要一些运行环境，这和 [从源代码安装](/zh-cn/install-from-source) 相同，如果你还没有设置好，可以先阅读那个章节。
+首先你需要一些运行环境，这和 [从源代码安装]({{< relref "from-source.zh-cn.md" >}}) 相同，如果你还没有设置好，可以先阅读那个章节。
```
Comment (tboerger) [inline content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: Without testing I would guess something like `{{< relref "../installation/from-source.zh-cn.md" >}}`. This version is just wrong as you can see on the docs page.

Comment (lunny) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: ^

Comment (luffyke) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: @tboerger i have tested it and result is expected before pr.

Comment (tboerger) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: @lunny i don't understand why you anyway linked to that page?

Comment (luffyke) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: @tboerger as from-source.zh-cn.md is unique in this project, `{{< relref "from-source.zh-cn.md" >}}` would point to `content/doc/installation/from-source.zh-cn.md`, and this fix with the expected result in my local workspace, `{{< relref "../installation/from-source.zh-cn.md" >}}` is incorrect, i think we can use `({{< relref "doc/installation/from-source.zh-cn.md" >}})`(tested normally in my local).

**Comment by bkcsoft** [comment]
> The local server results in correct link, but the docs page link results in <broken-link>

@tboerger that seems like a config-error for our træfik/caddy/hugo-setup (whatever we use) though 😉

**Comment by tboerger** [comment]
Traefik and caddy don't really change something like that. Feels more like some Hugo inconsistency

**Comment by luffyke** [comment]
Hello, is this pr need to backport v1.0 or not? as seems there is a hugo issue would cause 404 error.

**Comment by lunny** [comment]
@tboerger Why remove the backport/v1.0?

**Comment by luffyke** [comment]
May i know if this pr need backport to v1.0? thanks.

**Comment by lunny** [comment]
@tboerger here is OKay. @luffyke please send a PR to v1.0. Thanks.

**Pull request merged**